src/jvmTest/kotlin/DataClassTests.kt

```diff
@@ -0,0 +1,56 @@
+import data.CardType
+import data.LineItem
+import data.PurchaseType
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import kotlin.test.DefaultAsserter.assertEquals
+import kotlin.test.DefaultAsserter.assertTrue
+
+class DataClassTests {
+
+    private var item1: LineItem? = null;
+    private var expectedToString = ""
+    @BeforeEach
+    fun initVars(){
+        item1 = LineItem(1.11, 2.22, "test","1/2/23","Amazon", CardType.AH, PurchaseType.PURCHASE)
+        expectedToString = "LineItem(totalTaxable=1.11, totalNonTaxable=2.22, description=test, date=1/2/23, vendor=Amazon, cardType=AH, purchaseType=PURCHASE)"
+    }
+    @Test
+    fun lineItemCreate(){
+        assertEquals("Create Failed", expectedToString, item1.toString())
+    }
+
+    @Test
+    fun lineItemCopy(){
+        var item2 = item1?.copy()
+        assertEquals("Copy Failed", expectedToString, item2.toString())
+    }
+
+    @Test
+    fun lineItemNonTaxableZero(){
```

Comment:
Going forward with unit testing I would really like to see us implement a `Given_When_Then()` style for naming our tests.

For example I'd probably name this test something along the lines of
```suggestion
    fun lineItemNonTaxable_ZeroValue_LineItemCreationSuccessful(){
```

Reply:
It can get wordy, yes, but I think it is worth it to set up this system now so that we know exactly what our tests are trying to accomplish.

Reply:
noted and implemented